Formula/z/zsh-completions.rb

```diff
@@ -49,8 +49,8 @@ def caveats
       Additionally, if you receive "zsh compinit: insecure directories" warnings when attempting
       to load these completions, you may need to run these commands:
 
-        chmod go-w '#{HOMEBREW_PREFIX}/share'
-        chmod -R go-w '#{HOMEBREW_PREFIX}/share/zsh'
+        chmod go-w "#{HOMEBREW_PREFIX}/share"
+        chmod -R go-w "#{HOMEBREW_PREFIX}/share/zsh"
```

Comment:
what is the issue with the single quote?

Reply:
Well, it prints `chmod go-w '${HOMEBREW_PREFIX}/share'` and that does not evaluate `${HOMEBREW_PREFIX}` because its a literal quote.